hw/net/Makefile.objs

```diff
@@ -37,5 +37,6 @@ obj-$(CONFIG_ETSEC) += fsl_etsec/etsec.o fsl_etsec/registers.o \
 			fsl_etsec/rings.o fsl_etsec/miim.o
 
 common-obj-$(CONFIG_ROCKER) += rocker/rocker.o rocker/rocker_fp.o \
-                               rocker/rocker_desc.o rocker/rocker_world.o \
-                               rocker/rocker_of_dpa.o
+								rocker/rocker_desc.o rocker/rocker_world.o \
```

Comment:
My bad.. I had tab-stop set to 4 spaces.

On Sun, Apr 19, 2015 at 9:27 PM, Scott Feldman <notifications@github.com>
wrote:

> In hw/net/Makefile.objs
> <https://github.com/pbhide/qemu-rocker/pull/1#discussion_r28663511>:
>
> > @@ -37,5 +37,6 @@ obj-$(CONFIG_ETSEC) += fsl_etsec/etsec.o fsl_etsec/registers.o \
> >  			fsl_etsec/rings.o fsl_etsec/miim.o
> >
> >  common-obj-$(CONFIG_ROCKER) += rocker/rocker.o rocker/rocker_fp.o \
> > -                               rocker/rocker_desc.o rocker/rocker_world.o \
> > -                               rocker/rocker_of_dpa.o
> > +								rocker/rocker_desc.o rocker/rocker_world.o \
>
> Strange tab issue here...not sure what changed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pbhide/qemu-rocker/pull/1/files#r28663511>.
>